src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -1379,6 +1398,79 @@ mod test {
         check_truncate_data(tests);
     }
 
+    #[test]
+    fn test_truncate_decimal() {
+        let tests = vec![
```

Comment:
More corner cases please.

Reply:
thanks, I found some corner case issues, will checked and fixed later

Reply:
Hi, I add thress maximum  number test cases , do you have any other corner cases to suggest, thanks?

Reply:
LGTM

Reply:
Thank you for committing.

Reply:
Please take a look at the ci test. Use `cargo fmt` please.

Reply:
sorry, forgot to run `cargo fmt` before commit